indra/resources/entity_hierarchy.rdf

```diff
@@ -120,6 +120,9 @@
 <http://identifiers.org/hgnc.symbol/APOE> <http://sorger.med.harvard.edu/indra/relations/partof> <http://sorger.med.harvard.edu/indra/entities/HDL> .
 <http://identifiers.org/hgnc.symbol/ARAF> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/RAF> .
 <http://identifiers.org/hgnc.symbol/AREG> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/EGFR_ligand> .
+<http://identifiers.org/hgnc.symbol/ARHGDIA> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/RhoGDI> .
+<http://identifiers.org/hgnc.symbol/ARHGDIB> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/RhoGDI> .
+<http://identifiers.org/hgnc.symbol/ARHGDIG> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/RhoGDI> .
```

Comment:
I'm messing with update_resources and running into some issues. Will skip over this one until it's figured out. Probably also has to do with the version of bioentities.